### PR TITLE
Add asyncSuffix to webpack configuration

### PR DESCRIPTION
**Pull request by pavel-sindelka**

### CODE REVIEW

File: packages/react-union-scripts/scripts/webpack.config.js [hunk 1]
```diff
@@ -118,6 +119,20 @@ const getCommonConfig = ({ outputMapper }) => ({
 	},
 });
 
+const resolveAsyncSuffix = (asyncSuffix) => {
```
Comment (tommmyy) [inline packages/react-union-scripts/scripts/webpack.config.js]: Would be great if you use R.cond and `R_.isString`, `R_.isArray` and `R_.constructRegExp`

**Comment by pavel-sindelka** [comment]
#46

File: packages/react-union-scripts/scripts/webpack.config.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const ManifestPlugin = require('webpack-manifest-plugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const path = require('path');
```
Comment (tommmyy) [inline packages/react-union-scripts/scripts/webpack.config.js]: Just import:

```js
const { resolve } = require('path')
```
And you solve the naming problem with R.path.

**Comment by pavel-sindelka** [comment]
#46 @tommmyy

File: union.config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = ({
```
Comment (tommmyy) [inline union.config.js]: Remove this file

**Comment by tommmyy** [comment]
Thank you for your work!

**Pull request merged**